utils/cardUtils/storageHelpers.js

```diff
@@ -12,7 +12,8 @@ function setItem(key, value) {
 
 // eslint-disable-next-line no-unused-vars
 function getItem(key) {
-  let value;
+  // eslint-disable-next-line no-var
+  var value;
   try {
     const jsonValue = window.sessionStorage.getItem(key);
```

Comment:
shouldn't this be `var` also?

Reply:
It seems that here `try - catch` catch this error here. I'll change it anyway.